README.md

```diff
@@ -15,18 +15,7 @@ Simple LDAP server with Vagrant and Puppet as proof-of-concept of these technolo
 * READ SECTION BELOW
 * Visit phpLDAPadmin at http://localhost:8888/phpldapadmin in a browser.
 * If you want to use ApacheDirectoryStudio, use server at localhost:4444.
-
-### Important: service upstart broken ###
-For some reason the service decided to not start automatically in the last update. This means that to have the programs running, 
-one must ssh into the machine, change directory, and manually invoke node:
-
-* vagrant ssh
-* cd /srv/oms-core/lib
-* node server.js &
-* cd /srv/oms-profiles-module/lib
-* node server.js &
-
-they will start normally, but when invoked with the upstart script nope.. (maybe the fault is of "forever")
+* Maybe you will have to create the folders oms-events, oms-cron and oms-token-master in /ignore
```

Comment:
cool, can you do it please? the empty folders are not tracked so I added a .delme empty file in there, just do the same for the new folders